### PR TITLE
[CTOR-151] Adding missing packages

### PR DESCRIPTION
**Pull request by sdepassio**
… (only for debian bullseye)

## Description

Adding missing packages for openwsman:
- sblim-sfcc (only for debian bullseye)
- libwsman1 (same version as perl-openwsman)

### CODE REVIEW

**Review by omercier — APPROVED** [review]
Works with Alma8, Alma9 and Debian11

File: .github/workflows/perl-openwsman.yml [hunk 1]
```diff
@@ -138,8 +179,7 @@ jobs:
           rpm_gpg_signing_passphrase: ${{ secrets.RPM_GPG_SIGNING_PASSPHRASE }}
 
       # set condition to true if artifacts are needed
-      - if: ${{ false }}
-        name: Upload package artifacts
+      - name: Upload package artifacts
```
Comment (sc979) [inline .github/workflows/perl-openwsman.yml:182]: @kduret, shouldn't we keep this condition to avoid to always upload the artifact ?

Comment (kduret) [reply .github/workflows/perl-openwsman.yml:182]: Yes indeed

File: dependencies/perl-openwsman/libwsman.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    dst: "@USRLIB@"
+    file_info:
+      mode: 0644
+
+overrides:
+  rpm:
+    depends:
+      - sblim-sfcc
+    provides:
+      - libwsman1
```
Comment (kduret) [inline dependencies/perl-openwsman/libwsman.yaml:36]:
```suggestion
```

File: dependencies/perl-openwsman/libwsman.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  deb:
+    depends:
+      - sblim-sfcc
+    provides:
+      - libwsman1
```
Comment (kduret) [inline dependencies/perl-openwsman/libwsman.yaml:41]:
```suggestion
```

File: dependencies/perl-openwsman/perl-openwsman.yaml [hunk 4]
```diff
@@ -2,37 +2,38 @@ name: "@NAME@"
...
   rpm:
+    depends:
+      - libwsman1
     provides:
       - perl(openwsman)
   deb:
+    depends:
+      - libwsman1
     provides:
       - libopenwsman-perl
```
Comment (kduret) [inline dependencies/perl-openwsman/perl-openwsman.yaml:38]:
```suggestion
```

**Review by omercier — APPROVED** [review]

**Review by sc979 — APPROVED** [review]

**Review by lucie-dubrunfaut — APPROVED** [review]

**Pull request merged**